pytac/cothread_cs.py

```diff
@@ -23,13 +24,65 @@ def get(self, pv):
         Returns:
             float: Represents the current value of the given PV.
         """
-        return caget(pv)
+        try:
```

Comment:
According to the documentation, caget could return any scalar type so I thought it would be desirable to specify that further, to float since all our current PVs return floats currently anyway. In relation to suppressing the error and returning None, that is the standard for get_multiple style requests so that if one fails it doesn't scrap the whole lot and so I extrapolated and applied that to all methods.